core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/ProfileMain.scala

```diff
@@ -78,7 +80,15 @@ object ProfileMain extends Logging {
 
     val profiler = new Profiler(hadoopConf, appArgs, enablePB)
     if (driverLog.nonEmpty) {
-      profiler.profileDriver(driverLog, eventLogFsFiltered.isEmpty)
+      val dLogPath = new org.apache.hadoop.fs.Path(driverLog)
```

Comment:
@amahussein Regarding the comment in issue description,
> Furthermore, the class is not using hadoop API to load the files, which means that only files on local path can be processed.

Is this referring to the use case when the driver logs can be located in cloud storage directories (`s3`, `dbfs`), similar to what we have for event logs? 

In that case, we might have to use a similar approach as processing event logs

Reply:
Yes, this is exactly what this issue is about.
It does not make sense to me that we process eventlogs on remote storage but we are asking the user to download the driver-logs locally.